ui/app/templates/components/client-node-row.hbs

```diff
@@ -16,9 +16,9 @@
 <td data-test-client-datacenter>{{node.datacenter}}</td>
 <td data-test-client-volumes>{{if node.hostVolumes.length node.hostVolumes.length}}</td>
 <td data-test-client-allocations>
-  {{#if node.allocations.isPending}}
+  {{#if node.runningAllocations.isPending}}
```

Comment:
`runningAllocations` is not a relationship and will therefore not have an `isPending` property. This should still be `allocations`.

Reply:
ah - good catch - i'll fix it in a follow up PR and update some tests too.  Thanks!

Reply:
eep thanks for catching that, @DingoEatingFuzz 😳